R/write_tests_xlsx.R

```diff
@@ -34,7 +34,7 @@ write_tests_xlsx <- function(comparison_data, sheet_name) {
   }
 
   # add a new sheet for tests
-  sheet_name_dated <- paste0(sheet_name, format(Sys.Date(), "_%d_%b"))
+  sheet_name_dated <- paste0(year, "_", sheet_name, format(Sys.Date(), "_%d_%b"))
```

Comment:
I think glue makes this more readable.
```suggestion
  sheet_name_dated <- glue::glue("{year}_{sheet_name}_{format(Sys.Date(), '%d_%b')}")
```